### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [r0x0r/pywebview#1007] [Q&A] [answered]
Hi all.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by r0x0r** [comment, marked as answer]
Yes, it is.

**Reply by ghost** [reply]
Hi r0x0r.

> Yes, it is.

thank you for feedback. I will close this issue, you clarified my doubt.

*Have a great day, and thank you for your patience in answering my question.*
